tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/util/SQLDBUtils.java

```diff
@@ -18,6 +18,7 @@
 package org.ballerinalang.test.util;
```

Comment:
IINM, none of the tests that use this class seem to be running. Shall we create an issue to check if they are still valid/belong in the lang repo and enable/remove these.

Reply:
Ack. Will create an issue for this.